composables/auth.ts

```diff
@@ -78,7 +78,7 @@ export const useAuthStore = defineStore('auth', () => {
    * @param password password of the user, min length of 6
    * @returns ValidationError | undefined
    */
-  async function register(firstName: string, lastName: string, email: string, password: string) {
+  async function register(firstName: string, lastName: string, email: string, password: string, confirm_password: string) {
```

Comment:
As discussed in our earlier conversations, I don't think we will need `confirm_password`